src/main/resources/assets/gregtech/lang/en_us.lang

```diff
@@ -1300,18 +1300,19 @@ cover.fluid_regulator.transfer_mode.description=§eTransfer Any§r - in this mod
 cover.fluid_regulator.supply_exact=Supply Exact: %s
 cover.fluid_regulator.keep_exact=Keep Exact: %s
 
-cover.machine_controller.title=Machine Controller Settings
-cover.machine_controller.normal=Normal
```

Comment:
This lang entry is used in some other covers, like the Advanced Energy Detector. So removing it causes there to be a missing lang entry for that cover.